spdx-utils/src/main/kotlin/Extensions.kt

```diff
@@ -124,6 +124,12 @@ fun String.isSpdxExpression(): Boolean =
 fun String.isSpdxExpressionOrNotPresent(): Boolean =
     SpdxConstants.isNotPresent(this) || isSpdxExpression()
 
+/**
+ * Return this string lower-cased except for the first character which is upper-cased.
+ */
+fun String.titlecase() =
+    lowercase().replaceFirstChar { if (it.isLowerCase()) it.titlecase(Locale.getDefault()) else it.toString() }
```

Comment:
Refer to other remark with regards to usage of the default locale.

Reply:
See the discussion above.